backend/src/main/java/com/trainme/app/user/controller/UserProfileRestController.java

```diff
@@ -0,0 +1,36 @@
+package com.trainme.app.user.controller;
+
+import com.trainme.app.user.controller.exceptions.EntityNotFoundException;
+import com.trainme.app.user_profile.entity.UserProfile;
+import com.trainme.app.user_profile.service.UserProfileService;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class UserProfileRestController {
```

Comment:
It looks like a duplicate of `com.dmytrochernykh.trainme.user.controller.UserProfileRestController`.